libs/designsystem/checkbox/src/checkbox.component.ts

```diff
@@ -51,9 +60,59 @@ export class CheckboxComponent {
 
   onChecked(checked: boolean): void {
     this.checked = checked;
+    this._onChange(this.checked);
     this.checkedChange.emit(this.checked);
+    this._onTouched();
   }
 
   // IDs used for a11y labelling
   _labelId = UniqueIdGenerator.scopedTo('kirby-checkbox-label').next();
+
+  // Initialize empty ControlValueAccessor callback functions
+  // eslint-disable-next-line no-empty-function
+  private _onChange: (value: boolean) => void = () => {};
+  // eslint-disable-next-line no-empty-function
+  private _onTouched = () => {};
+
+  /**
+   * Sets the select's value. Part of the ControlValueAccessor interface
```

Comment:
Copy pasted from somehere? ;)
```suggestion
   * Sets the checkbox's value. Part of the ControlValueAccessor interface
```